README.md

```diff
@@ -30,23 +30,30 @@ The external scaler calls Cosmos DB APIs to estimate the amount of changes pendi
 
 > :warning: **Caution:** The [Java SDK v2](https://github.com/Azure/azure-cosmosdb-java) client library uses a different naming convention for lease documents inside the lease container. This makes it incompatible with [.NET SDK v3](https://github.com/Azure/azure-cosmos-dotnet-v3), the one that the external scaler depends on to estimate the pending changes on change feeds. Hence, if you have a Java-based target consumer application, your change feeds would be having lease documents with incompatible IDs, and the external scaler would be unable to detect any pending change remaining to be consumed. Consequently, it will scale down your application to `minReplicaCount` if defined in the `ScaledObject` or to zero instances.
 
-### Deploy KEDA
+### Deploy KEDA and External Scaler
 
-Follow one of the steps on [Deploying KEDA](https://keda.sh/docs/deploy/) documentation page to deploy KEDA on your Kubernetes cluster.
+1. Add and update Helm chart repo.
 
-### Deploy External Scaler
+    ```shell
+    helm repo add kedacore https://kedacore.github.io/charts
+    helm repo update
+    ```
 
-Clone the repository and change to root directory of the cloned repo. Apply the configuration from file `deploy/deploy-scaler.yaml`. It deploys the external scaler `Deployment` resource to the `keda` namespace. It also creates a `Service` resource, that KEDA will send requests to, to fetch the Cosmos DB metrics.
+1. Install KEDA Helm chart (*or follow one of the other installation methods on [KEDA documentation](https://keda.sh/docs/deploy)*).
 
-Before applying the configuration, please update the image path in the YAML file if you want to deploy the scaler from a non-`experimental` image. Similarly, you can also update the name and namespace of the external scaler `Deployment` and `Service`.
+    ```shell
+    helm install keda kedacore/keda --namespace keda --create-namespace
+    ```
 
-```text
-kubectl apply filename=deploy/deploy.yaml
-```
+1. Install 'Cosmos DB scaler' Helm chart. This will deploys the external scaler `Deployment` resource to the `keda` namespace. It also creates a `Service` resource, that KEDA will send requests to, to fetch the Cosmos DB metrics.
```

Comment:
I'd remove all the details as this is implementation details, but just a suggestion
```suggestion
1. Install the Azure Cosmos DB external scaler Helm chart.
```